repository/s.json

```diff
@@ -86,6 +86,22 @@
 			]
 		},
 		{
+			"name": "Sasha",
+			"details": "https://github.com/Kristinita/SashaSublime",
+			"labels": ["language syntax"],
+			"homepage": "https://github.com/Kristinita/SashaSublime",
+			"author": "Sasha Chernykh",
+			"readme": "https://github.com/Kristinita/SashaSublime/blob/master/README.md",
+			"issues": "https://github.com/Kristinita/SashaSublime/issues",
```

Comment:
`readme` and `issues` are also set to their default values based on `details`, so you may omit them.